tools/ui-components/src/alert/alert.stories.tsx

```diff
@@ -14,49 +14,65 @@ const story = {
 
 const Template: Story<AlertProps> = args => <Alert {...args} />;
 
-export const Basic = Template.bind({});
-Basic.args = {
```

Comment:
This pr should be good to go as soon we remove the `Dismissable` variation. Please feel free to discuss the matter further if necessary if you don't find the point made above convincing.

Reply:
I will get this pr in for now so we could keep moving forward, and we could further discuss the `Dismissable` variation if needed in details.

Reply:
Here is my proposal:
- We will remove dismissability-related props from the `Alert` component, to make it non-dismissable
- Build a `Flash` component with props for dismiss action. The props should be required to give a clear distinction between the two

As for the implementation approach, we can either:
1. Create a `BaseAlert` component that has the alert styling applied and supports all props. Then have `Alert` built upon it but prohibit dismissability-related props (via the props interface, thanks to TypeScript). `Flash` would also be built upon `BaseAlert`, but have stricter props interface to enforce the dismissability; or
2. Basically just duplicate the code and keep `Alert` and `Flash` not related to each other. 

(2) is simpler to build while (1) is a bit more involved, but with mobile and dark theme support, I think going with (1) might be worth it as it allows us to keep the styling between the two components consistent.

---
Here is the interfaces I have in mind:

```ts
type AlertVariant = 'success' | 'info' | 'warning' | 'danger';
type FlashVariant = AlertVariant;

interface BaseAlertProps {
  children: React.ReactNode;
  className?: string;
  variant: AlertVariant;
  dismissLabel?: string;
  onDismiss?: () => void;
} 

interface AlertProps {
  children: React.ReactNode;
  className?: string;
  variant: AlertVariant;
}

interface FlashProps {
  children: React.ReactNode;
  className?: string;
  variant: FlashVariant; // This can totally be AlertVariant, but the naming might be confusing
  dismissLabel: string; // <- required prop
  onDismiss: () => void; // <- required prop
}
```